### DISCUSSION TITLE
Do Seed Values Persist on Shutdown for Google Collab?

### DISCUSSION THREAD

**Discussion by JoeyGorombey** [lllyasviel/Fooocus#2056] [Q&A] [answered]
For instance, if I have a seed, XZYW, is XYZW retrievable upon starting the next instance of Fooocus? Does Fooocus persist seeds with respect to the images those seeds point to?

**Comment by eddyizm** [comment, marked as answer]
I'm not sure about the paid version but the free one is uses an ephemeral compute engine, nothing is saved unless you have your googe drive mounted but seeds themselves are not save anywhere besides the private in general.
